src/components/MenuBar/index.tsx

```diff
@@ -85,16 +96,42 @@ const MenuBar = () => {
                 </Box>
 
                 <Divider orientation={"vertical"}/>
-                <NavigationBar/>
+                <Box className={"menu-bar-calendar-container"}>
+                    <MenuBarIconButton
+                        disabled={isDisabled(uiState, UI_ELEMENT.NAVIGATION_BAR)}
+                        tooltipPlacement={"bottom-start"}
```

Comment:
"bottom"? if we don't have any specific reason to use "bottom-start" instead of just "bottom" like most of the other buttons?